src/app/workflow/launch-third-party/launch-third-party.component.ts

```diff
@@ -237,12 +237,16 @@ export class LaunchThirdPartyComponent extends Base implements OnChanges, OnInit
     return `Export this workflow version to ${platform}.`;
   }
 
+  private isGitHubWorkflow(): boolean {
+    return this.workflow && this.workflow.gitUrl && this.workflow.gitUrl.startsWith('git@github.com');
+  }
+
   private terraTooltip(hasContent: boolean, hasFileImports, platform: string): string {
```

Comment:
platform is a variable in the methodname and text, but the method is called terraTooltip